plugin.video.montreal.greek-tv/resources/lib/youtube.py

```diff
@@ -0,0 +1,352 @@
+# -*- coding: utf-8 -*-
+
+'''
+    Montreal Greek TV Add-on
+    Author: greektimes
+
+        License summary below, for more details please read license.txt file
+
+        This program is free software: you can redistribute it and/or modify
+        it under the terms of the GNU General Public License as published by
+        the Free Software Foundation, either version 2 of the License, or
+        (at your option) any later version.
+        This program is distributed in the hope that it will be useful,
+        but WITHOUT ANY WARRANTY; without even the implied warranty of
+        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+        GNU General Public License for more details.
+        You should have received a copy of the GNU General Public License
+        along with this program.  If not, see <http://www.gnu.org/licenses/>.
+'''
+from __future__ import absolute_import, division, unicode_literals
+
+import re, json
+from resources.lib.compat import urlparse, parse_qs, quote_plus, range
+from resources.lib import workers, control, directory
+import requests
+from CommonFunctions import parseDOM
+
+
+def yt_resolve(setting='yt_resolve'):
+
+    resolve_bool = control.setting(setting) == 'true' and any(
+        [
+            control.condVisibility('System.HasAddon(script.module.{0})'.format('urlresolver')),
```

Comment:
remove any references to script.module.urlresolver or similar. At least you can use youtube_dl.

Reply:
ok.